contracts/StrategyCurveTwoPool.sol

```diff
@@ -0,0 +1,372 @@
+      uint256 _loss,
+      uint256 _debtPayment
+    )
+  {
+    // harvest our rewards from the gauge
+    gauge.claim_rewards();
+
+    uint256 crvBalance = crv.balanceOf(address(this));
+
+    // if we claimed any CRV, then sell it
+    if (crvBalance > 0) {
+      _sell(crvBalance);
+      uint256 usdtBalance = usdt.balanceOf(address(this));
+      uint256 usdcBalance = usdc.balanceOf(address(this));
+      curve.add_liquidity([usdcBalance, usdtBalance], 0);
+    }
+
+    // debtOustanding will only be > 0 in the event of revoking or if we need to rebalance from a withdrawal or lowering the debtRatio
+    if (_debtOutstanding > 0) {
+      uint256 stakedBal = stakedBalance(); // Balance of gauge tokens by depositing curve LP tokens into the curve gauge
+      if (stakedBal > 0) {
+        // don't bother withdrawing if we don't have staked funds
+        gauge.withdraw(Math.min(stakedBal, _debtOutstanding));
+      }
+      uint256 _withdrawnBal = balanceOfWant(); // Balance of curve LP token
+      _debtPayment = Math.min(_debtOutstanding, _withdrawnBal);
+    }
+
+    // serious loss should never happen, but if it does (for instance, if Curve is hacked), let's record it accurately
+    uint256 assets = estimatedTotalAssets();
+    uint256 debt = vault.strategies(address(this)).totalDebt;
+
+    // if assets are greater than debt, we have a profit
+    if (assets > debt) {
+      _profit = assets.sub(debt);
+      uint256 _wantBal = balanceOfWant();
+      if (_profit.add(_debtPayment) > _wantBal) {
+        // this should only be hit following donations to strategy
+        liquidateAllPositions();
+      }
+    }
+    // if assets are less than debt, we have a loss
+    else {
+      _loss = debt.sub(assets);
+    }
+
+    // we're done harvesting, so reset our trigger if we used it
+    forceHarvestTriggerOnce = false;
+  }
+
+  // Sell our CRV in Uniswap V3 for our `targetToken`, USDT or USDC
+  function _sell(uint256 _amount) internal {
+    // gets multi-hop path for uniswap v3 swaps
+    bytes memory path = abi.encodePacked(
+      address(crv),
+      crvToWethSwapFee,
+      wethAddress,
+      wethToTargetSwapFee,
+      targetTokenAddress
+    );
+
+    router.exactInput(
+      ISwapRouter.ExactInputParams(
+        path, // multi-hop path
+        address(this),
+        now,
+        _amount,
+        0 // amountOutMinimum
+      )
+    );
+  }
+
+  /* ========== SETTERS ========== */
+
+  // These functions are useful for setting parameters of the strategy that may need to be adjusted.
+  // Set optimal token to sell harvested funds for depositing to Curve.
+  // Default is USDC, but can be set to USDT as needed by strategist or governance.
+  function setTargetToken(uint256 _target) external onlyVaultManagers {
+    require(_target == 0 || _target == 1); // dev: not a valid index => use 0: USDT 1: USDC
+    targetTokenAddress = address(usdt);
+    if (_target == 1) {
+      targetTokenAddress = address(usdc);
+    }
+  }
```

Comment:
```suggestion
  function setTargetToken(uint256 _target) external onlyVaultManagers {
    require(_target == 0 || _target == 1); // dev: not a valid index => use 0: USDT 1: USDC
    
    if (_target == 1) {
      targetTokenAddress = address(usdc);
    } else {
      targetTokenAddress = address(usdt);
  }
```

One assignment instead of 2.

Reply:
nice! thanks!